### PR TITLE
Dundee workshops

### PR DESCRIPTION
**Pull request by jburel**
Add events
Link to registration page to be updated when ready (nothing for June)
cc @pwalczysko @jrswedlow

### CODE REVIEW

**Comment by sbesson** [comment]
@jburel planning a redeployment of www tomorrow morning. Let me know if you want to get this reviewed and bumped in the playbook.

**Comment by jburel** [comment]
@sbesson it will be good to have it included.
I have updated the link

File: index.html [hunk 1]
```diff
@@ -12,7 +12,8 @@ <h1 class="hero-main-header small-12">Next Workshop</h1>
                     </div>
                     <h2 class="hero-main-header small-12">Upcoming Workshops</h2>
                     <div align="center">
-                    <p align="left" class="small-10 small-offset-1 medium-6 medium-offset-3"></p>
+                    <p align="left" class="small-10 small-offset-1 medium-6 medium-offset-3">08-09 April in Dundee -- <a href="{{ site.baseurl }}/events/omero-users-workshop-dundee-april-2019.html">programme</a></p>
+                    <p align="left" class="small-10 small-offset-1 medium-6 medium-offset-3">10-11 June in Dundee -- <a href="{{ site.baseurl }}/events/omero-users-workshop-dundee-june-2019.html">programme</a></p></p>
```
Comment (mtbc) [inline index.html]: two `</p>` seems odd

**Comment by will-moore** [comment]
Looks good 👍

**Comment by pwalczysko** [comment]
Links work and text is okay.

But,
1 .layout-readability problem. The blue links "program" are very hard to read, as does the white text - this is because they are on the "bridge" and, additionally, the sky and the water are also blue...:)

screen shot 2019-02-26 at 10 17 49


2. layout problem - see in the screenshot above, the "Next workshops" are adjusted to the left border, but the "Upcoming..." are central ?  This could be unified.

**Comment by jburel** [comment]
@pwalczysko point 2 will require a bit of rework. I will leave it as it is for now.
The color has been changed as discussed.

**Comment by pwalczysko** [comment]
Looks much better, thanks

**Pull request merged**